.github/workflows/esp32_build.yml

```diff
@@ -181,14 +195,14 @@ jobs:
           update-alternatives --query python
 
           rm -rf /usr/local/bin/cmake
-          sudo apt remove --purge --auto-remove cmake
-          sudo apt update && \
-          sudo apt install -y software-properties-common lsb-release && \
-          sudo apt clean all
+          sudo apt-get remove --purge --auto-remove cmake 2>/dev/null >/dev/null
+          sudo apt-get update && \
+          sudo apt-get install -y software-properties-common lsb-release && \
+          sudo apt-get clean all
           wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
           sudo apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
-          sudo apt update
-          sudo apt install cmake
+          sudo apt-get update 2>/dev/null >/dev/null
```

Comment:
We shouldn't be throwing this oputput away

Reply:
its the 3rd 'apt-get update' call.  ... by then if there were network issues, its already failed on one of the earlier ones.  :-)